src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.js

```diff
@@ -150,20 +146,6 @@ function FloatingActionButtonAndPopover(props) {
         }
     };
 
-    useEffect(() => {
-        const navigationState = props.navigation.getState();
-        const routes = lodashGet(navigationState, 'routes', []);
-        const currentRoute = routes[navigationState.index];
-        if (currentRoute && ![NAVIGATORS.CENTRAL_PANE_NAVIGATOR, SCREENS.HOME].includes(currentRoute.name)) {
-            return;
-        }
-        if (lodashGet(props.demoInfo, 'money2020.isBeginningDemo', false)) {
-            return;
-        }
-        Welcome.show({routes, showCreateMenu});
```

Comment:
We're no longer showing FAB menu after first signup?

Reply:
Correct, it's just this popup now